build.zig

```diff
@@ -124,7 +124,17 @@ pub fn build(b: *zbs.Builder) !void {
         rivertile.install();
     }
 
-    b.installFile("protocol/river-layout-v3.xml", "share/river/river-layout-v3.xml");
+    {
+        const file = try fs.path.join(b.allocator, &[_][]const u8{ b.cache_root, "river-protocols.pc" });
+        const pkgconfig_file = try std.fs.cwd().createFile(file, .{});
+
+        const writer = pkgconfig_file.writer();
+        try writePkgconfigFile(writer, b.install_prefix);
+        defer pkgconfig_file.close();
+
+        b.installFile("protocol/river-layout-v3.xml", "share/river-protocols/river-layout-v3.xml");
+        b.installFile(file, "lib/pkgconfig/river-protocols.pc");
```

Comment:
wayland-protocols puts this in `$PREFIX/share/pkgconfig/`, maybe we should do the same?

Reply:
pkg-config man page say they are both correct, I personally have `~/.local/lib/pkgconfig` in my $PKG_CONFIG_PATH` so that's why and looks like most packages I have use `lib`

Reply:
I suspect most packages use `lib` because they are using their `.pc` file to give information about files they have installed in `lib` while `wayland-protocols` and we don't install anything to `lib`, only to `share`.

Reply:
From the man page "the default is `libdir/pkgconfig:datadir/pkgconfig`" so maybe they just put it in the first one, anyway if you prefer share I'll use share, no problem

Reply:
cool, let's go with share then.